web/Services/AdminService.cs

```diff
@@ -44,18 +48,68 @@ private async Task<List<UserDbModel>> GetCurrentAdmins()
 
     private async Task<List<UserDbModel>> GetPendingAdmins()
     {
-        List<UserDbModel> PendingAdmins = await _dbContext.Users.Where(u => u.IsAdmin == false && u.HasRequestedAdmin).ToListAsync();
+        List<UserDbModel> PendingAdmins =
+            await _dbContext.Users.Where(u => u.IsAdmin == false && u.HasRequestedAdmin).ToListAsync();
         return PendingAdmins;
     }
 
     public async Task<(List<UserDbModel> CurrentAdmins, List<UserDbModel> PendingAdmins)> GetAdminLists()
     {
         return (await GetCurrentAdmins(), await GetPendingAdmins());
     }
-    
+
     public async Task<List<LandlordDbModel>> GetUnapprovedLandlords()
     {
-        List<LandlordDbModel> UnapprovedLandlords = await _dbContext.Landlords.Where(u => u.CharterApproved == false).ToListAsync();
+        List<LandlordDbModel> UnapprovedLandlords =
+            await _dbContext.Landlords.Where(u => u.CharterApproved == false).ToListAsync();
         return UnapprovedLandlords;
     }
+
+    public UserDbModel? FindUserByLandlordId(int landlordId)
+    {
+        return _dbContext.Users.SingleOrDefault(u => u.LandlordId == landlordId);
+    }
+    
+    public string FindExistingInviteLink(int landlordId)
```

Comment:
It might be slightly more intuitive to return `null` in the case where there's no link, then the return type here would be `string?`

Reply:
yeah sure done!